### PR TITLE
 9995 DOD: Replace DateInput and DatePickerComponent with DateSelector

### PR DESCRIPTION
**Pull request by rachelschneiderman**

### CODE REVIEW

File: web-client/integration-tests/admissionsClerkPractitionerJourney.test.ts [hunk 1]
```diff
@@ -99,7 +99,7 @@ describe('admissions clerk practitioner journey', () => {
   loginAs(cerebralTest, 'admissionsclerk@example.com');
   admissionsClerkAddsPractitionerEmail(cerebralTest);
 
-  // show Practitioners only, and not case-users (bug ref: #8081)
+  // // show Practitioners only, and not case-users (bug ref: #8081)
```
Comment (mmarcotte) [inline web-client/integration-tests/admissionsClerkPractitionerJourney.test.ts]: probably unnecessary; might be able to just get rid of the line entirely at this point.

File: web-client/src/views/CaseDetail/DeficiencyStatisticsForm.tsx [hunk 2]
```diff
@@ -91,30 +96,27 @@ export const DeficiencyStatisticsForm = connect(
...
 
-              {form.yearOrPeriod === 'Period' && (
-                <FormGroup errorText={validationErrors.lastDateOfPeriod}>
-                  <DateInput
-                    id="last-date-of-period"
-                    label="Last date of period"
-                    names={{
-                      day: 'lastDateOfPeriodDay',
+              {/*
+                                day: 'lastDateOfPeriodDay',
```
Comment (mmarcotte) [inline web-client/src/views/CaseDetail/DeficiencyStatisticsForm.tsx]: is this left included for any specific reason?

File: web-client/src/views/Practitioners/PractitionerForm.tsx [hunk 3]
```diff
@@ -375,32 +371,29 @@ export const PractitionerForm = connect(
...
                 </div>
               </div>
             </div>
           </div>
         </div>
+
         <div className="grid-row margin-bottom-4">
           <div className="grid-col-12">
-            <h2>Practitioner Notes</h2>
+            <h2>Practitioner Notesss</h2>
```
Comment (mmarcotte) [inline web-client/src/views/Practitioners/PractitionerForm.tsx]: Are you sssure about this one? 😉 🐍

File: web-client/src/views/StampMotion/ApplyStamp.tsx [hunk 4]
```diff
@@ -348,30 +345,23 @@ export const ApplyStamp = connect(
                         stipulated decision by:
                       </label>
                     </div>
-                    <DateInput
-                      className="display-inline-block padding-0 margin-left-5"
+                    {/*
+                      // disabled={!form.dueDateMessage}
+                     */}
```
Comment (mmarcotte) [inline web-client/src/views/StampMotion/ApplyStamp.tsx]: any good reason to keep this?

**Comment by mmarcotte** [comment]
wow, what a massive detangling and headache relief!

**Comment by mmarcotte** [comment]
-3,804 lines!!

File: web-client/integration-tests/admissionsClerkPractitionerJourney.test.ts (hunk 1 above)
Comment (rachelschneiderman) [reply web-client/integration-tests/admissionsClerkPractitionerJourney.test.ts]: Good point, thanks.

File: web-client/src/views/CaseDetail/DeficiencyStatisticsForm.tsx (hunk 2 above)
Comment (rachelschneiderman) [reply web-client/src/views/CaseDetail/DeficiencyStatisticsForm.tsx]: Nope! Just forgot to delete it, thanks.

File: web-client/src/views/StampMotion/ApplyStamp.tsx (hunk 4 above)
Comment (rachelschneiderman) [reply web-client/src/views/StampMotion/ApplyStamp.tsx]: Nope! Another oopsies

**Review by mmarcotte — APPROVED** [review]

**Pull request merged**